avm/res/web/site/config/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 The latest version of the changelog can be found [here](https://github.com/Azure/bicep-registry-modules/blob/main/avm/res/web/site/config/CHANGELOG.md).
 
+
+## 0.1.1
+
+### Changes
+
+- Added Test Case / Usage Example
```

Comment:
Hey @tsc-buddy,

thanks for adding the test case 🙏 Just one comment:

This should be updated (in this and the other Changelogs) to

```suggestion
- Recompiled templates with latest Bicep version `0.38.33.2757`
```

Reason: The changelog should focus on what change about the module. An update to the test is more a validation / readme change that won't impact a consumer of the module. I hope that makes sense (:

Alternatively, you could also have only regenerated the readme, which would not publish a new module version at all, hence not requireing updates to the changelog.